modules/admin-ui/src/main/resources/web/admin/apps/content-manager/js/app/create/NewContentDialogList.ts

```diff
@@ -20,7 +20,7 @@ module app.create {
             });
         }
 
-        private notifySelected(listItem: NewContentDialogListItem) {
```

Comment:
If we don't need in duplicated method in MostPopularItemList, then it 'notifySelected' could be stay private